lib/rules/comma-dangle.js

```diff
@@ -128,10 +132,14 @@ module.exports = function(context) {
         var sourceCode = context.getSourceCode();
         var trailingToken = sourceCode.getTokenAfter(lastItem);
         if (trailingToken.value !== ",") {
-            context.report(
-                lastItem,
-                lastItem.loc.end,
-                MISSING_MESSAGE);
+            context.report({
+                node: lastItem,
+                loc: lastItem.loc.end,
+                message: MISSING_MESSAGE,
+                fix: function(fixer) {
+                    return fixer.insertTextAfter(lastItem, ",");
```

Comment:
It does seem like eslint should make another full pass if at least one fix was applied to a file. Any other way seems overly complicated. Each of the results of these passes needs to be stored, so that they can be compared to detect non-terminating fix loops, which represent conflicting rules with fixes, which arguably should/could be detected before the fix situation arises.